docs/source/guides/development/getting_started.rst

```diff
@@ -0,0 +1,312 @@
+Getting Started with Characters
+===============================
+
+
+* `Side Channels <#a-note-about-side-channels>`_
+* `Running an Ethereum Node <#running-an-ethereum-node>`_
+*  Connecting Nucypher to an Etheruem Node <#connecting-to-the-nucypher-network>`_
+*  Alice: Grant Access to a Secret <#alice-grant-access-to-a-secret>`_
+    * `Setup Alice <#setup-alice>`_
+    * `Grant <#grant>`_
+* `Enrico: Encrypt a Secret <#enrico-encrypt-a-secret>`_
+    * `Setup Enrico <#setup-enrico>`_
+    * `Encrypt <#encrypt>`_
+* `Bob: Decrypt a Secret <#bob-decrypt-a-secret>`_
+    * `Setup Bob <#setup-bob>`_
+    * `Join a Policy <#join-a-policy>`_
+    * `Decrypt <#retrieve-and-decrypt>`_
+
+
+A Note about Side Channels
+--------------------------
+
+The NuCypher network does not store or handle an application's data; instead - it manages access *to* application data.
+Management of encrypted secrets and public keys tends to be highly domain-specific - The surrounding architecture
+will vary greatly depending on the throughput, sensitivity, and sharing cadence of application secrets.
+In all cases, NuCypher must be integrated with a storage and transport layer in order to function properly.
+Along with the transport of ciphertexts, a nucypher application also needs to include channels for Alice and Bob 
+to discover each other's public keys, and provide policy encrypting information to Bob and Enrico.
+
+Side Channel Application Data
+-----------------------------
+
+
+* Secrets:
+  * Message Kits - Encrypted Messages, or "Ciphertexts"
```

Comment:
This section isn't formatted properly. Trying adding an empty line between 1st-level bullets and second-level bullets.